app/controllers/events_controller.rb

```diff
@@ -101,6 +101,9 @@ def ranking
   end
 
   def overview
+    if @event.type == 'League'
+      redirect_to @event, error: 'No overview available for leagues.'
```

Comment:
Please make sure to actually use the localization you created (`overview.not_available_for_leagues`) ;-)